frontend/src/components/Rounting.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react';
+import '../App.global.scss';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import SignIn from '../screens/SignIn';
```

Comment:
Can we name these `pages` not `screens` ? It is common convention, screens are usually referred to in mobile